### ISSUE TITLE
Cachet is not working if it doesn't have a internet connection

### ISSUE THREAD

**Issue by Mebus** [cachethq/Cachet#896]
We use Cachet to inform our customers when the Internet connection fails.

Unfortunately we discovered, that it tries to load Emojis from http://api.github.com/emojis and thus fails with an Error 500 when it doesn't have a internet connection. 

![failed1cut](https://cloud.githubusercontent.com/assets/271557/9248906/3e5a835a-41c0-11e5-92b1-733aa441513f.png)

This means that if your internet connection fails or api.github.com fails also Cachet fails, which doesn't make it very robust.

It also includes Google fonts form an external URL (which doesn't have effects on the server-side, but might lead to problems on the client side.)

Please fix these issues to make Cachet work without a internet connection.

**Comment by jbrooksuk** [comment]
1. You can manually remove emoji by going to `./config/app.php` and commenting out the `EmojiServiceProvider` line.
2. Google Fonts won't prevent your page from loading, you'll just not have the font.

**Comment by Mebus** [comment]
Referring to 1:

I already did this, but:

I think it should automatically disable emoji, if emoji is not working, because having eomjis is not important. Nobody will know about this problem until the internet connection fails. And then you might have other things to do than fixing Emojis!

Referring to 2:

I'm not really shure about this. Might have to test this again.

**Comment by rtrauntvein** [comment]
I tend to agree with @Mebus here - Emoji is not really critical, and in lots of (enterprise) status page use cases may even be considered inappropriate to use Emoji.  When users are looking for a status of a critical service, the last thing they need to see is a emoticon :)

**Comment by jbrooksuk** [comment]
Usage of emoticons is completely optional due to the syntax required to make them `:smile:` for instance.

**Comment by rtrauntvein** [comment]
Right I understand that - It is just a feature that in my opinion doesn't add much value in most use cases, but can cause the whole site to break if it is unreachable

**Comment by Mebus** [comment]
@jbrooksuk Even if you don't use them anywhere, the site will fail. ;-)

**Comment by jbrooksuk** [comment]
@Mebus yeah, so for now you can remove `EmojiServiceProvider`.

@cachethq/owners perhaps we should have a config for this?

**Comment by TheoBearman** [comment]
@jbrooksuk I think that would be sensible:

**Comment by GrahamCampbell** [comment]
We DO have config for this...

**Comment by GrahamCampbell** [comment]
The service provider is in the config...

**Comment by GrahamCampbell** [comment]
This is a non-issue.

**Comment by Mebus** [comment]
Sorry, but then the default config is broken.

**Comment by jbrooksuk** [comment]
> Sorry, but then the default config is broken.

The default config works perfectly for the way Cachet is intended to be used. If you want use it differently, then you have to modify the config I'm afraid.

**Comment by GrahamCampbell** [comment]
The default config is not broken. Your server setup is broken.

**Comment by GrahamCampbell** [comment]
If you want to use that server setup, you need to adjust the config. We're not going to spoil everyone else's experience just because of your server setup, sorry.

**Comment by Mebus** [comment]
Ok. So Cachet depends on api.github.com and will die with an Error 500 if that site is down. Great. 

I can't understand why anbody would want this. But it's your project...

**Comment by GrahamCampbell** [comment]
> Ok. So Cachet depends on https://api.github.com and will die with an Error 500 if that site is down. Great.

No, it uses the cache.

**Comment by GrahamCampbell** [comment]
GitHub is unlikely to be down for more than 12 hours...

**Comment by GrahamCampbell** [comment]
Hmmm, maybe we do have an issue here.

**Comment by GrahamCampbell** [comment]
This is the wrong repo to discuss this. Please open an issue on our emoji repo.

**Comment by Mebus** [comment]
It's easy to reproduce:

Edit /etc/hosts

add

127.0.0.1       api.github.com

run

php artisan config:cache

=>

<code>
Error creating resource: [message] fopen(): SSL operation failed with code 1. OpenSSL Error messages: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed [file] /var/www/status/Cachet/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 244 [message] fopen(): Failed to enable crypto [file] /var/www/status/Cachet/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 244 [message] fopen(https://api.github.com/emojis): failed to open stream: operation failed [file] /var/www/status/Cachet/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 244
</code>

Mebus

**Comment by jbrooksuk** [comment]
1. Status pages are pretty much ALWAYS intended to be public facing, so you'll have an internet connection.
2. This is no different to you editing the `.env` file in the first place, it's still a config file.

Regardless, we're going to handle this error better.

**Comment by Mebus** [comment]
Oh dear. Wouldn't it be simple to catch the error, ignore all Emojis and write a warning to the log file?

If our company loses the link to the internet, what would a status page do for me which is hosted in the internet, when I want to use it to inform the 200 people in our local network?

**Comment by jbrooksuk** [comment]
This will be fixed in 1.2

**Comment by Mebus** [comment]
Thanks. Because it's such a nice software and I cannot understand why we're having such a long discussion at all. For me it is important, that it works as robust as possible.

**Comment by jbrooksuk** [comment]
This is now fixed in 1.2

**Comment by rauppe31** [comment]
Hello

I currently have the same problem. My system is not able to download https://api.github.com/emojis

```
root@status:~# curl https://api.github.com/emojis
curl: (35) Unknown SSL protocol error in connection to api.github.com:443
root@status:~# wget https://api.github.com/emojis
--2018-01-26 07:50:11--  https://api.github.com/emojis
Resolving api.github.com (api.github.com)... 192.30.253.117, 192.30.253.116
Connecting to api.github.com (api.github.com)|192.30.253.117|:443... connected.
GnuTLS: Error in the pull function.
Unable to establish SSL connection.
```

The cachethq page is now displaying an error 500. The logfile outputs this:

```
Next GuzzleHttp\Exception\RequestException: Error creating resource: [message] fopen(): SSL: Connection reset by peer
[file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php
[line] 324
[message] fopen(): Failed to enable crypto
[file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php
[line] 324
[message] fopen(https://api.github.com/emojis): failed to open stream: operation failed
[file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php
[line] 324 in /var/www/html/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:52
Stack trace:
#0 /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php(67): GuzzleHttp\Exception\RequestException::wrapException(Object(GuzzleHttp\Psr7\Request), Object(RuntimeException))
#1 /var/www/html/vendor/guzzlehttp/guzzle/src/PrepareBodyMiddleware.php(37): GuzzleHttp\Handler\StreamHandler->__invoke(Object(GuzzleHttp\Psr7\Request), Array)
#2 /var/www/html/vendor/guzzlehttp/guzzle/src/Middleware.php(30): GuzzleHttp\PrepareBodyMiddleware->__invoke(Object(GuzzleHttp\Psr7\Request), Array)
#3 /var/www/html/vendor/guzzlehttp/guzzle/src/RedirectMiddleware.php(70): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Request), Array)
#4 /var/www/html/vendor/guzzlehttp/guzzle/src/Middleware.php(59): GuzzleHttp\RedirectMiddleware->__invoke(Object(GuzzleHttp\Psr7\Request), Array)
#5 /var/www/html/vendor/guzzlehttp/guzzle/src/HandlerStack.php(67): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Request), Array)
#6 /var/www/html/vendor/guzzlehttp/guzzle/src/Client.php(277): GuzzleHttp\HandlerStack->__invoke(Object(GuzzleHttp\Psr7\Request), Array)
#7 /var/www/html/vendor/guzzlehttp/guzzle/src/Client.php(125): GuzzleHttp\Client->transfer(Object(GuzzleHttp\Psr7\Request), Array)
#8 /var/www/html/vendor/guzzlehttp/guzzle/src/Client.php(131): GuzzleHttp\Client->requestAsync('get', Object(GuzzleHttp\Psr7\Uri), Array)
#9 /var/www/html/vendor/guzzlehttp/guzzle/src/Client.php(89): GuzzleHttp\Client->request('get', 'https://api.git...', Array)
#10 /var/www/html/vendor/alt-three/emoji/src/EmojiServiceProvider.php(81): GuzzleHttp\Client->__call('get', Array)
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Cache/Repository.php(292): AltThree\Emoji\EmojiServiceProvider->AltThree\Emoji\{closure}()
#12 [internal function]: Illuminate\Cache\Repository->remember('emoji', 10080, Object(Closure))
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(296): call_user_func_array(Array, Array)
#14 /var/www/html/vendor/alt-three/emoji/src/EmojiServiceProvider.php(83): Illuminate\Cache\CacheManager->__call('remember', Array)
#15 /var/www/html/bootstrap/cache/compiled.php(1353): AltThree\Emoji\EmojiServiceProvider->AltThree\Emoji\{closure}(Object(Illuminate\Foundation\Application), Array)
#16 /var/www/html/bootstrap/cache/compiled.php(1309): Illuminate\Container\Container->build(Object(Closure), Array)
#17 /var/www/html/bootstrap/cache/compiled.php(1874): Illuminate\Container\Container->make('AltThree\\Emoji\\...', Array)
#18 /var/www/html/bootstrap/cache/compiled.php(1410): Illuminate\Foundation\Application->make('AltThree\\Emoji\\...')
#19 /var/www/html/bootstrap/cache/compiled.php(1387): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#20 /var/www/html/bootstrap/cache/compiled.php(1373): Illuminate\Container\Container->getDependencies(Array, Array)
#21 /var/www/html/bootstrap/cache/compiled.php(1309): Illuminate\Container\Container->build('AltThree\\Emoji\\...', Array)
#22 /var/www/html/bootstrap/cache/compiled.php(1874): Illuminate\Container\Container->make('AltThree\\Emoji\\...', Array)
#23 /var/www/html/vendor/graham-campbell/markdown/src/MarkdownServiceProvider.php(150): Illuminate\Foundation\Application->make('AltThree\\Emoji\\...')
#24 /var/www/html/bootstrap/cache/compiled.php(1353): GrahamCampbell\Markdown\MarkdownServiceProvider->GrahamCampbell\Markdown\{closure}(Object(Illuminate\Foundation\Application), Array)
#25 /var/www/html/bootstrap/cache/compiled.php(1309): Illuminate\Container\Container->build(Object(Closure), Array)
#26 /var/www/html/bootstrap/cache/compiled.php(1874): Illuminate\Container\Container->make('markdown.enviro...', Array)
#27 /var/www/html/bootstrap/cache/compiled.php(1556): Illuminate\Foundation\Application->make('markdown.enviro...')
#28 /var/www/html/vendor/graham-campbell/markdown/src/MarkdownServiceProvider.php(167): Illuminate\Container\Container->offsetGet('markdown.enviro...')
#29 /var/www/html/bootstrap/cache/compiled.php(1353): GrahamCampbell\Markdown\MarkdownServiceProvider->GrahamCampbell\Markdown\{closure}(Object(Illuminate\Foundation\Application), Array)
#30 /var/www/html/bootstrap/cache/compiled.php(1309): Illuminate\Container\Container->build(Object(Closure), Array)
#31 /var/www/html/bootstrap/cache/compiled.php(1874): Illuminate\Container\Container->make('markdown', Array)
#32 /var/www/html/bootstrap/cache/compiled.php(1556): Illuminate\Foundation\Application->make('markdown')
#33 /var/www/html/bootstrap/cache/compiled.php(6297): Illuminate\Container\Container->offsetGet('markdown')
#34 /var/www/html/bootstrap/cache/compiled.php(6283): Illuminate\Support\Facades\Facade::resolveFacadeInstance('markdown')
#35 /var/www/html/bootstrap/cache/compiled.php(6317): Illuminate\Support\Facades\Facade::getFacadeRoot()
#36 /var/www/html/app/Composers/AppComposer.php(29): Illuminate\Support\Facades\Facade::__callStatic('convertToHtml', Array)
#37 [internal function]: CachetHQ\Cachet\Composers\AppComposer->compose(Object(Illuminate\View\View))
#38 /var/www/html/bootstrap/cache/compiled.php(14628): call_user_func_array(Array, Array)
#39 [internal function]: Illuminate\View\Factory->Illuminate\View\{closure}(Object(Illuminate\View\View))
#40 /var/www/html/bootstrap/cache/compiled.php(10124): call_user_func_array(Object(Closure), Array)
#41 /var/www/html/bootstrap/cache/compiled.php(14641): Illuminate\Events\Dispatcher->fire('composing: feed...', Array)
#42 /var/www/html/bootstrap/cache/compiled.php(15081): Illuminate\View\Factory->callComposer(Object(Illuminate\View\View))
#43 /var/www/html/bootstrap/cache/compiled.php(15066): Illuminate\View\View->renderContents()
#44 /var/www/html/bootstrap/cache/compiled.php(16309): Illuminate\View\View->render()
#45 /var/www/html/bootstrap/cache/compiled.php(15805): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#46 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View), 200, Array)
#47 /var/www/html/bootstrap/cache/compiled.php(6329): Illuminate\Routing\ResponseFactory->make(Object(Illuminate\View\View), 200, Array)
#48 /var/www/html/vendor/roumen/feed/src/Roumen/Feed/Feed.php(279): Illuminate\Support\Facades\Facade::__callStatic('make', Array)
#49 /var/www/html/app/Http/Controllers/FeedController.php(93): Roumen\Feed\Feed->render('rss')
#50 /var/www/html/app/Http/Controllers/FeedController.php(68): CachetHQ\Cachet\Http\Controllers\FeedController->feedAction(Object(CachetHQ\Cachet\Models\ComponentGroup), true)
#51 [internal function]: CachetHQ\Cachet\Http\Controllers\FeedController->rssAction(Object(CachetHQ\Cachet\Models\ComponentGroup))
#52 /var/www/html/bootstrap/cache/compiled.php(9419): call_user_func_array(Array, Array)
#53 /var/www/html/bootstrap/cache/compiled.php(9481): Illuminate\Routing\Controller->callAction('rssAction', Array)
#54 /var/www/html/bootstrap/cache/compiled.php(9461): Illuminate\Routing\ControllerDispatcher->call(Object(CachetHQ\Cachet\Http\Controllers\FeedController), Object(Illuminate\Routing\Route), 'rssAction')
#55 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#56 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#57 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#58 /var/www/html/bootstrap/cache/compiled.php(9938): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#59 /var/www/html/bootstrap/cache/compiled.php(9462): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#60 /var/www/html/bootstrap/cache/compiled.php(9449): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(CachetHQ\Cachet\Http\Controllers\FeedController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'rssAction')
#61 /var/www/html/bootstrap/cache/compiled.php(8517): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'CachetHQ\\Cachet...', 'rssAction')
#62 /var/www/html/bootstrap/cache/compiled.php(8504): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#63 /var/www/html/bootstrap/cache/compiled.php(8218): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#64 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#65 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#66 /var/www/html/app/Http/Middleware/ReadyForUse.php(40): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#67 [internal function]: CachetHQ\Cachet\Http\Middleware\ReadyForUse->handle(Object(Illuminate\Http\Request), Object(Closure))
#68 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#69 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#70 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#71 /var/www/html/bootstrap/cache/compiled.php(3225): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#72 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#73 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#74 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#75 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#76 /var/www/html/bootstrap/cache/compiled.php(13400): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#77 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#78 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#79 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#80 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#81 /var/www/html/bootstrap/cache/compiled.php(11943): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#82 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#83 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#84 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#85 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#86 /var/www/html/bootstrap/cache/compiled.php(13139): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#87 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#88 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#89 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#90 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#91 /var/www/html/bootstrap/cache/compiled.php(13076): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#92 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#93 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#94 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#95 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#96 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#97 /var/www/html/bootstrap/cache/compiled.php(9938): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#98 /var/www/html/bootstrap/cache/compiled.php(8219): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#99 /var/www/html/bootstrap/cache/compiled.php(8210): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#100 /var/www/html/bootstrap/cache/compiled.php(8200): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#101 /var/www/html/bootstrap/cache/compiled.php(2419): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#102 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#103 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#104 /var/www/html/bootstrap/cache/compiled.php(3286): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#105 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#106 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#107 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#108 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#109 /var/www/html/vendor/fideloper/proxy/src/TrustProxies.php(56): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#110 [internal function]: Fideloper\Proxy\TrustProxies->handle(Object(Illuminate\Http\Request), Object(Closure))
#111 /var/www/html/bootstrap/cache/compiled.php(9953): call_user_func_array(Array, Array)
#112 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#113 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#114 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#115 /var/www/html/bootstrap/cache/compiled.php(9938): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#116 /var/www/html/bootstrap/cache/compiled.php(2366): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#117 /var/www/html/bootstrap/cache/compiled.php(2350): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#118 /var/www/html/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#119 {main} {"identification":{"id":"55119b41-73eb-4e05-b3b4-22e8cacbf5d6"}}
```

Can someone please check if this fix is still working?
